src/clientenvironment.cpp

```diff
@@ -237,7 +238,7 @@ void ClientEnvironment::step(float dtime)
 			pre_factor = 1.0 + (float)addp/100.0;
 		}
 		float speed = pre_factor * speed_diff.getLength();
-		if (speed > tolerance) {
+		if (speed > tolerance && itemgroup_get(lplayer->getCAO()->getGroups(), "immortal") == 0) {
```

Comment:
Line too long (>80 characters)

Reply:
also imho `!itemgroup_get(lplayer->getCAO()->getGroups(), "immortal")` would be nicer then comparing it with zero

Reply:
(80 columns with tab size 4)